README.md

```diff
@@ -0,0 +1,60 @@
+# Launch Checklist
+
+## Assignment Requirements
+
+For this "Launch Checklist Form" assignment, you will use a form submission and fetch request to accomplish the following:
+
+1. Validate the user responses with `preventDefault()` to ensure the following: 
+
+   a. The user has entered something for every field.
+   b. The user has entered string values for names and number for fuel and cargo levels.
+
+1. With validation, update a list of what is currently ready or not ready for the shuttle launch.
```

Comment:
Formating is off -> 

a and b should be on separate lines

Reply:
Done!